pallets/automation-time/src/lib.rs

```diff
@@ -173,12 +174,13 @@ pub mod pallet {
 		type DelegatorActions: DelegatorActions<Self::AccountId, BalanceOf<Self>>;
 
 		/// The overarching call type.
-		type Call: Parameter
+		type RuntimeCall: Parameter
```

Comment:
Rename `Call` to `RuntimeCall`.

Reply:
What is the reason of this type re-definition?